model/event/DeliveryCreatedEvent.php

```diff
@@ -19,11 +19,11 @@
  */
 namespace oat\taoDeliveryRdf\model\event;
 
```

Comment:
version change required please don't forget about upadte and manifest,
also please apply the mandatory list provided by @Babacooll